roadmap/implementors-guide/guide.md

```diff
@@ -989,12 +991,78 @@ Dispatch a `PovFetchSubsystemMessage(relay_parent, candidate_hash, sender)` and
 
+### Statement Gossip Subsystem
+
+The statement gossip subsystem is responsible for gossiping out and receiving [Signed Statements](#Signed-statement-type) from validators.
+
+#### Message Types
+
+This subsystem communicates with the overseer via [two message types](#Candidate-Backing-Subsystem-Messages), one for overseer -> subsystem and another for subsystem -> overseer. Currently this just differentiates between statements that the subsystem has been instructed to gossip out, and statements that the subsystem has received.
+
+#### Functionality
+
+The subsystem needs to contain a handle to a gossiping engine to gossip and recieve messages. This will be cloned into each job that is spawned. Apart from that, it also contains the general structures that all subsystems contain, e.g. channels to communicate with the overseer and handles to spawned jobs in order to shut them down.
+
+On `OverseerSignal::StartWork` it should:
+* Spawn a job and pass in `relay_parent`, a clone of the gossiping engine and a handle to a message validator.
+* Send out a neighbour packet with the `relay_parent` added to the list of accepted chain heads.
+
+On `OverseerSignal::StopWork` it should:
+* Stop the job via the job handle.
+* Send out a neighbour packet with the job's `relay_parent` removed.
+
+On `StatementGossipSubsystemMessageIn::StatementToGossip` it should:
+* Send the signed statement to the job running for the `relay_parent`.
+
+The statement gossip job needs to poll two seperate futures (as well as the exit signal):
+
+* A future that takes the passed-in statements, validates them and gossips them along with the `relay_parent` hash using the gossip engine.
+* A future that takes the messages that the gossip engine receives for the `relay_parent`, validates them and sends to the subsystem.
+
+#### Message Validation
+
+The following conditions needs to be met for an outgoing statement to be sent:
+
+* We must have 'peer knowledge' of what canidates the receiver knows about.
+* If the statement is `Valid` or `Invalid`, the receiver of the statement must know about the candidate that the hash refers to.
+* The statement signature must be valid.
+
+The following conditions need to be met for an incoming statement to be accepted:
+
+* The sender of the statement has to be in the validator set for the relay block.
+* If the statement is `Valid` or `Invalid`, then the candidate that the hash refers to must be known.
+* As only `MAX_CHAIN_HEADS` (currently set to 5) competing parachain heads are accepted at a time, if the statement is `Seconded`, then there must be less than `MAX_CHAIN_HEADS` other candidate receipts for the parachain block.
+* The statement signature must be valid.
+
+I have a basic implementation of this code on the [`ashley-test-statement-gossip-subsystem`](https://github.com/paritytech/polkadot/tree/ashley-test-statement-gossip-subsystem) branch.
```

Comment:
In case you would like to have input on this, would you mind opening up a pull request? That would make discussions a lot easier.